src/main/java/games/strategy/thread/ThreadPool.java

```diff
@@ -53,7 +53,7 @@ public void waitForAll() {
       } catch (final InterruptedException e) {
         Thread.currentThread().interrupt();
       } catch (final ExecutionException e) {
-        ClientLogger.logError(e.getCause());
+        ClientLogger.logError("Threading execution exception: " + e.getMessage(), e);
```

Comment:
Ditto on the `ExecutionException` comment above.

Reply:
Updated in: c47f11775, error message also updated to use the message from the cause exception as well.